### DISCUSSION TITLE
Websocket - notifications only for conversations that assigned to the agent

### DISCUSSION THREAD

**Discussion by Naomi1900** [chatwoot/chatwoot#5877] [Self hosted] [answered]
When I open websocket for agent, I get notifications for all conversations in the inbox,
Is there a way to get the push notifications only for the conversations that assigned to this agent ?

**Comment by sojan-official** [comment, marked as answer]
Not at the moment. All updates required by the agent dashboard are pushed through the same WebSocket connection. You could filter the events and handle only the ones where the conversation assignee is that specific agent.

**Reply by Naomi1900** [reply]
In this way there will be a lot of unnecessary traffic. 
For example if there are 50 active chats, and each one is assigned to a different agent, then each  agent will get a lot of events that is not relevant to hem.

If I still want to filter the messages on the server side , Where do I change this in the code?

**Reply by sojan-official** [reply]
ref: https://github.com/chatwoot/chatwoot/blob/develop/app/listeners/action_cable_listener.rb
